xmppserver/src/main/java/org/jivesoftware/openfire/commands/admin/user/UserProperties.java

```diff
@@ -97,10 +101,12 @@ private void populateResponseFields(DataForm form, List<String> accounts) {
     }
```

Comment:
When requesting a list of users, each field has a list of values. 
Where the Name is missing, the list gets an empty string appended. 
Where the email is missing, the list isn't appended to at all.
This could lead to some consistency problems.

Reply:
I've pushed a change that defaults the email to an empty string when its missing so that the value sets should be the same length.